indico/core/db/sqlalchemy/locations.py

```diff
@@ -248,3 +248,48 @@ def location_data(self):
             return {'source': data_source, 'venue': data_source.venue, 'room': data_source.room,
                     'room_name': data_source.room_name, 'venue_name': data_source.venue_name,
                     'address': data_source.address, 'inheriting': self.inherit_location}
+
+    @property
+    def location_data_for_ui(self):
```

Comment:
This is a serialization method. I think @ThiefMaster's opinion is that we put them in `utils.py` while I think we should have some `@serializer` decorator that allows us to quickly add a base one with flavors if needed in the same fashion we have `@locator.ui`/`@locator.registrant`. Maybe we are in time to add it for this relase?